src/main/java/com/project/bookstore/model/Order.java

```diff
@@ -0,0 +1,63 @@
+package com.project.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "orders")
+@Setter
+@Getter
+@RequiredArgsConstructor
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "status", columnDefinition = "varchar")
+    private Status status;
+    @Column(name = "order_date", nullable = false)
+    private LocalDateTime orderDate;
+    @Column(nullable = false)
+    private BigDecimal total;
+    @Column(name = "shipping_address", nullable = false)
+    private String shippingAddress;
+    @OneToMany(mappedBy = "order", fetch = FetchType.LAZY)
+    private Set<OrderItem> orderItems = new HashSet<>();
+
+    public enum Status {
+        NEW("new"),
+        PAID("paid"),
+        PROCESSING("processing"),
+        DELIVERED("delivered"),
+        CANCELLED("cancelled");
+
+        private final String name;
+
+        Status(String name) {
+            this.name = name;
+        }
+
+        @Override
+        public String toString() {
+            return name;
+        }

```

Comment:
enum.name().toLowerCase() can be used instead
```suggestion
        NEW,
        PAID,
        PROCESSING,
        DELIVERED,
        CANCELLED
```